test/lib/k8s/job.js

```diff
@@ -0,0 +1,305 @@
+const chai = require("chai");
```

Comment:
we don't need a separate line for this now since the `chai-as-promised` setup is done in a bootstrap file.  we can import `expect` like this now:

```js
const { expect } = require("chai");
```